test_macros/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
```

Comment:
@brokad we might want to move this funky little library to its own public crate and repo as a developer tool (in a separate PR) :smile: